airrohr-firmware/airrohr-firmware.ino

```diff
@@ -2575,7 +2577,10 @@ static unsigned long sendCFA(const String &data, const int pin, const __FlashStr
 		sensor_readings.print(", ");
 		sensor_readings.print(sensorname);
 		sensor_readings.println("/t");
-		sum_send_time = sendData(LoggerCFA, data_CFA, pin, HOST_CFA, URL_CFA);
+
+		if(WIFI_enabled){
+			sum_send_time = sendData(LoggerCFA, data_CFA, pin, HOST_CFA, URL_CFA);
+		}
 	}
 
 	return sum_send_time;
```

Comment:
@MCLEANS in the event that WIFI_enabled is False, sum_send_time would not be defined.